R/dm.R

```diff
@@ -568,6 +568,27 @@ format.zoomed_df <- function(x, ..., n = NULL, width = NULL, n_extra = NULL) {
   NextMethod()
 }
 
+print.dm_keys <- function(x, ..., n = 3, width = NULL, n_extra = NULL) {
```

Comment:
I think this doesn't have an effect. Can we remove it without breaking functionality?